erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -398,9 +459,16 @@ void ExternalOutput::queueData(char* buffer, int length, packetType type){
           context_->oformat->audio_codec = AV_CODEC_ID_PCM_MULAW;
         }
     }
-    if (needToSendFir_ && videoSourceSsrc_) {
+
+    timeval time;
+    gettimeofday(&time, NULL);
```

Comment:
You'll want to use a monotonic timer here; I realize other parts of the code don't (including parts I use) but it's more stable.

```
uint64_t getMonotonicTimeInUSec() {
    struct timespec ts;
    clock_gettime(CLOCK_MONOTONIC, &ts);
    return (ts.tv_sec * uint64_t(1000000)) + (ts.tv_nsec / uint64_t(1000));
}
```